src/main/java/uk/gov/justice/probation/courtcasematcher/service/MatcherService.java

```diff
@@ -41,14 +48,17 @@ public void match(Case incomingCase) {
     }
 
     private Mono<CourtCase> newMatchedCaseOf(Case incomingCase) {
+        final String courtCode = incomingCase.getBlock().getSession().getCourtCode();
+        final String caseNo = incomingCase.getCaseNo();
         return offenderSearchRestClient.search(incomingCase.getDef_name(), incomingCase.getDef_dob())
                 .map(searchResponse -> {
-                    log.info(String.format("Match results for caseNo: %s, courtCode: %s - matchedBy: %s, matchCount: %s", incomingCase.getCaseNo(), incomingCase.getBlock().getSession().getCourtCode(), searchResponse.getMatchedBy(), searchResponse.getMatches() == null ? "null" : searchResponse.getMatches().size()));
+                    log.info(String.format("Match results for caseNo: %s, courtCode: %s - matchedBy: %s, matchCount: %s", caseNo, courtCode, searchResponse.getMatchedBy(), searchResponse.getMatches() == null ? "null" : searchResponse.getMatches().size()));
                     return searchResponse;
                 })
                 .filter(searchResponse -> searchResponse.getMatchedBy() == MatchType.ALL_SUPPLIED)
                 .map(SearchResponse::getMatches)
                 .flatMap(matches -> {
+                    postMatches(courtCode, caseNo, matches, MatchType.ALL_SUPPLIED);
```

Comment:
Couldn't recall whether we we're doing just exact matches at this stage or all matches but think it makes sense to do just exact as you have. We must make sure there are tickets to follow up with other match types though.

Reply:
Yes - I passed the MatchType in this way because that's what was being filtered. Expect it will change as searches start.